docs/TRANSLATIONS.MD

```diff
@@ -0,0 +1,58 @@
+# 🐍 🌍 🚀Translation Release Strategy
+Thanks for your interest in translating the micro:bit Python Editor. Translations for all the Micro:bit Educational Foundation’s editors and apps are contributed by an amazing team of volunteers and we’d love for you to be a part of this too. 
+
+This document sets out the strategy and process for how the Foundation manages the translation of the Python Editor hosted at python.microbit.org
```

Comment:
To be fair, this lists how we manage the translations in this repository, which is upstream of the one hosted at microbit.org. Readers might assume this document doesn't apply to this specific project codebase?

Reply:
Not sure how to rephrase this so it covers that. maybe this is best to decide on a call

Reply:
I think is just enough if we don't mention the hosted editor. We can just say this, which is 100% true, and applies to both repos.

```
This document sets out the strategy and process for how the Foundation manages the translation of the Python Editor.
```